custom_components/zwave_mqtt/__init__.py

```diff
@@ -130,6 +131,12 @@ def async_value_changed(value):
             value.label,
             value.value,
         )
+        # Handle a scene activation message
+        if value.command_class in [
+            "COMMAND_CLASS_SCENE_ACTIVATION",
+            "COMMAND_CLASS_CENTRAL_SCENE",
+        ]:
+            handle_scene_activated(hass, value)
 
```

Comment:
Add a return. No further processing needed. 
```suggestion
            return

```